static/js/server_events_dispatch.js

```diff
@@ -372,13 +375,6 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
                         return;
                     }
 
-                    event.user_ids.forEach((user_id) => {
-                        if (!peer_data.remove_subscriber(sub.stream_id, user_id)) {
-                            blueslip.warn("Cannot process peer_remove event.");
```

Comment:
It seems like we're removing error checking for cases where the user wasn't already marked as subscribed in the migration to bulk queries.  I'm a bit hesitant about that, in that the most likely way we'd find a bug in this logic is via that mechanism, but it's also not logic that's super likely to have bugs.

Reply:
Yeah, I think it would be hard to break this code given the improved frontend tests in this PR and the coverage we now get from `check-schema` tool, but I am gonna modify this commit to be just more of a pure code move for now.